cyences_app_for_splunk/default/data/ui/views/cs_splunk_admin_reports.xml

```diff
@@ -16,15 +16,27 @@
     <latest></latest>
     <sampleRatio>1</sampleRatio>
   </search>
+  <search id="base_search">
```

Comment:
Rename the id.
"base_search" is very generic, name something like "base_disk_search" or "base_usage_search".